lib/src/mount.rs

```diff
@@ -23,6 +23,14 @@ use serde::Deserialize;
 
 use crate::task::Task;
 
+/// Well known identifier for pid 1
+pub(crate) const PID1: Pid = const {
+    match Pid::from_raw(1) {
+        Some(v) => v,
+        None => panic!("Expected to parse pid1"),
+    }
```

Comment:
Tangential but this one is set to just be `const { Pid::from_raw(1).unwrap() }` in the next Rust which will be a big big improvement for `const`.